pennylane/templates/state_preparations/arbitrary_state_preparation.py

```diff
@@ -70,9 +71,11 @@ def vqe(weights):
     Args:
         weights (array[float]): The angles of the Pauli word rotations, needs to have length :math:`2^(n+1) - 2`
             where :math:`n` is the number of wires the template acts upon.
-        wires (List[int]): The wires on which the arbitrary unitary acts.
+        wires (List[int]): The wires on which the arbitrary unitary acts. Also accepts
```

Comment:
Instead of List[int] should we have Sequence[int]?

Reply:
Good point!

Reply:
```suggestion
        wires (Sequence[int]): The wires on which the arbitrary unitary acts. Also accepts
```